src/FedexRest/Services/Ship/Entity/Label.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace FedexRest\Services\Ship\Entity;
+
+class Label
+{
+    public ?string $imageType;
+    public ?string $labelStockType;
+
+    /**
+     * @param string  $imageType
+     * @return $this
+     */
+    public function setImageType(string $imageType):Label
```

Comment:
Super minor non-blocking nitpick. There is some inconsistency with the return type formatting in a few places here.   Might be worth cleaning up, now if it's convenient or in another effort, if not.

Reply:
Yes, the return types seem perfect now! Thanks for taking care of that. Sorry, I thought I'd left a comment earlier, but must have misfired.